pkgs/development/python-modules/sgmllib3k/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, lib
+}: buildPythonPackage rec {
+  pname = "sgmllib3k";
+  version = "1.0.0";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1s8jm3dgqabgf8x96931scji679qkhvczlv3qld4qxpsicfgns3q";
+  };
+
+  # N.B. Tests are completely broken. The package doesn't even ship with the
```

Comment:
Did you try the link from the upstream repo?

Reply:
What do you mean? I couldn't even find the upstream repo, only the PyPI tarball

Reply:
I meant to ask if we could fetch the source code from GitHub or somewhere else. The repo appears to be https://github.com/hsoft/sgmllib.

Reply:
I have switched the source to github, now the tests pass.

Reply:
Nice, thank you!